homeassistant/components/maxcube/climate.py

```diff
@@ -234,8 +235,8 @@ def set_preset_mode(self, preset_mode):
     def extra_state_attributes(self):
         """Return the optional state attributes."""
         if not self._device.is_thermostat():
-            return {}
-        return {ATTR_VALVE_POSITION: self._device.valve_position}
+            return {ATTR_BATTERY: self._device.battery}
+        return {ATTR_VALVE_POSITION: self._device.valve_position, ATTR_BATTERY: self._device.battery}
```

Comment:
This should not be added as an attribute but as a separate sensor entity.

Reply:
Would it be consistent then? Now for example valve_position also given as attribute. 
The change just extends it.
![image](https://user-images.githubusercontent.com/38714065/150502283-4b03c366-45cd-4b13-9e0b-91eefb923e8b.png)

Reply:
> Would it be consistent then? Now for example valve_position also given as attribute.
> The change just extends it.

This is how we used to do it. Just because we did it in the past, it doesn't mean we still do. This needs to become a separate sensor entity.

Reply:
I see. 
So, you propose to create a separate sensor for every Max device in integration. There are could be many of them, for example in large house it could be 20 and more themostats.
If we create a separate sensor for every thermostat it mean that we have to match battery to thermostat device anywhere (automation UI etc), right?
Would it be still possible to extend an attribute and keep it as I proposed for simplicity?

Reply:
> Would it be still possible to extend an attribute

No that isn't possible, this new attribute must be a separate entity.

Reply:
Then, it was done alredy in #58699